src/langchain_google_bigtable/common.py

```diff
@@ -12,15 +12,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from importlib.metadata import version
 from typing import Optional
 
 from google.cloud import bigtable
 
 default_client: Optional[bigtable.Client] = None
+PACKAGE_NAME = "langchain-google-bigtable-python"
+USER_AGENT = PACKAGE_NAME + "/" + version("langchain-google-bigtable")
 
 
-def get_default_client() -> bigtable.Client:
+def use_client_or_default(client: Optional[bigtable.Client]) -> bigtable.Client:
+    if client:
+        client._client_info.user_agent = (
+            (client._client_info.user_agent or "") + " " + USER_AGENT
```

Comment:
To prevent adding the same user agent, this logic may be better https://github.com/googleapis/langchain-google-datastore-python/blob/3a62380794ad567ff8e9c0d5d7054093fb54cdb4/src/langchain_google_datastore/document_loader.py#L160-L168

Reply:
Done